packages/runtime-web/src/api/device/clipboard/index.ts

```diff
@@ -0,0 +1,34 @@
+import get from 'lodash.get'
+import { copy } from './helper'
+
+const isSupportNavigatorClipboard = () =>
+  typeof navigator.clipboard === 'object' && window.isSecureContext
+// 查询用户是否配置强制使用 command 实现
+const requireUseCommand =
+  get(window.$MOR_APP_CONFIG, 'apis.clipboard') === 'command'
+
+const NOT_SUPPORT_CLIPBOARD = 'Your browser not support clipboard api!'
```

Comment:
Your browser not support clipboard api => Your browser doesn't support clipboard api.